src/Raven.Server/Documents/AttachmentsStorage.cs

```diff
@@ -997,5 +1033,26 @@ public static void AssertAttachments(BlittableJsonReaderObject document, Documen
             }
         }
 #endif
+        public void AssertAttachmentsFromReplication(BlittableJsonReaderObject document, AttachmentType type,string id, DocumentsOperationContext context)
+        {
+            var changeVector = type == AttachmentType.Document ? null : document.GetChangeVector();
```

Comment:
Is it supposed to work for attachments of revisions as well (`AttachmentType.Revision`)?

Reply:
it suppose to unless you know something i don't, what is the issue here?

Reply:
Just asking. I wondered why we pass `null` here for revisions - I saw your other comment about the key that we search on and understand now